.ruby-version

```diff
@@ -1 +1 @@
-2.6.2
+3.0.0
```

Comment:
Might be good to use the latest stable - 3.2.2

Reply:
@kieranj Yes, 3.0.0 was a sweet spot for other dependencies in alpine, but as the docker stuff is not included on this PR, I'll update it.